Doc/library/pdb.rst

```diff
@@ -529,6 +529,14 @@ by the local file.
 
    Quit from the debugger.  The program being executed is aborted.
 
+.. pdbcommand:: debug code
+
+   Enter a recursive debugger that steps through the code
```

Comment:
```Enter in a recursive debugger...``` ?

Reply:
This is a verbatim copy from docs string. I could be wrong here but "Enter in" doesn't sound better than "Enter"

Reply:
@csabella can you please help us on this one? Thanks :)))

Reply:
`Enter` works here.